xml/deployment_hwrecommend.xml

```diff
@@ -252,7 +252,8 @@
    <itemizedlist mark="bullet" spacing="normal">
     <listitem>
      <para>
-      CPU recommendations:
+      The following CPU recommendations accounts for all devices
```

Comment:
```suggestion
      The following CPU recommendations account for all devices
```

Reply:
Isn't it more that it's independent of usage of device?
So, "The following CPU recommendations account for devices independent of usage"?